src/Orleans.Core/Configuration/Options/TelemetryOptions.cs

```diff
@@ -0,0 +1,20 @@
+using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+
+namespace Orleans.Configuration.Options
+{
+    public class TelemetryOptions
+    {
+        internal IList<Type> Consumers { get; set; } = new List<Type>();
+    }
+
+    public static class TelemetryOptionsExtensions
+    {
+        public static TelemetryOptions AddConsumer<T>(this TelemetryOptions options) where T : ITelemetryConsumer
```

Comment:
I thought we had build warnings for lack of doc comments on public types

Reply:
Don't think that is active on core assembly.